backend/data_preparation/crawler/twitter_crawler.py

```diff
@@ -144,4 +156,5 @@ def fetch_status_id_from_db():
 if __name__ == '__main__':
     crawler = TweetCrawler(TweetExtractor(), TweetDumper())
 
-    crawler.start(['wildfire'], 100, fetch_from_db=True, end_clause=None)
+    # Make sure at most one is set True, either fetch_from_db or using_sample_API
```

Comment:
it's better if the code is testing for such case, you may raise an Error if both set to be True in runtime to alter the user.